### ISSUE TITLE
Wrong Captcha code after deserialization

### ISSUE THREAD

**Issue by antonmironovv** [telerik/kendo-ui-core#6614]
### Bug report

Although Captcha restores private fields from serialization logic text still can be regenerated due to the CharSet property setter.

Ticket Id: 1539844

Related to issue: [https://github.com/telerik/telerik-captcha/issues/1](https://github.com/telerik/telerik-captcha/issues/1)

### Environment

Kendo UI version: 2021.3.914
Browser: [all]

**Comment by Iankodj** [comment]
The Telerik.Web.Captcha package is built includes a fix about that in 1.0.1 version. Make sure to test it out using the updated version.

**Comment by Iankodj** [comment]
Example code that can be used to test serialization outside examples:

```C#
        static void Main(string[] args)
        {
            CaptchaImage newCaptcha = CaptchaHelper.GetNewCaptcha();
            newCaptcha.TextLength = 3;
            newCaptcha.TextColor = System.Drawing.Color.Black;
            newCaptcha.BackgroundColor = System.Drawing.Color.Red;
            newCaptcha.BackgroundNoise = CaptchaBackgroundNoiseLevel.None;
            newCaptcha.Width = 300;
            newCaptcha.Height = 80;
            CaptchaHelper.RenderCaptcha(newCaptcha).Save("C:\\Users\\idjemere\\source\\repos\\ConsoleApp1\\ConsoleApp1\\newCaptha.png", ImageFormat.Jpeg);

            var ms = CaptchaHelper.SpeakText(newCaptcha);
            using (FileStream file = new FileStream("C:\\Users\\idjemere\\source\\repos\\ConsoleApp1\\ConsoleApp1\\newCaptha.wav", FileMode.Create, FileAccess.Write))
            {
                ms.WriteTo(file);
            }
            ms.Close();

            string json = JsonConvert.SerializeObject(newCaptcha);
            CaptchaImage captcha = JsonConvert.DeserializeObject<CaptchaImage>(json);
            Console.WriteLine(newCaptcha.UniqueId + " | " + captcha.UniqueId);
            Console.WriteLine(newCaptcha.Text + " | " + captcha.Text);
            Console.WriteLine(newCaptcha.RenderedAt + " | " + captcha.RenderedAt);

            CaptchaHelper.RenderCaptcha(captcha).Save("C:\\Users\\idjemere\\source\\repos\\ConsoleApp1\\ConsoleApp1\\captha.png", ImageFormat.Jpeg);
           
            ms = CaptchaHelper.SpeakText(captcha);
            using (FileStream file = new FileStream("C:\\Users\\idjemere\\source\\repos\\ConsoleApp1\\ConsoleApp1\\captcha.wav", FileMode.Create, FileAccess.Write))
            {
                ms.WriteTo(file);
            }
            ms.Close();
        }
```
